### DISCUSSION TITLE
Entering Events

### DISCUSSION THREAD

**Discussion by fschwinger** [SmartDataAnalytics/OpenResearch#110] [Ideas]
When I tried out OpenResearch starting of January, after Wolfgang pointed it out in the i5 Doctoral Seminar at RWTH Aachen University, I added two conference events into the system.

See: https://www.openresearch.org/wiki/Special:Contributions/Felix

I like the idea of a crowdsourced conference tracking tool. One drawback here at the moment is that the conferences in my specific domains, Intelligent Transportation Systems, were not in the system at that time. Usually, I find the conferences in my domain using key papers of other authors. Wolfgang pointed to the proceedings title parser, however, I did not use it again so far.

Are there any plans on automating the addition of conference events into the system?

**Comment by WolfgangFahl** [comment]
My proposal would be to import all dblp conference series see https://confident.dbis.rwth-aachen.de/dblpconf/ Do you find your conference series there?

**Reply by fschwinger** [reply]
Yes, I found my conference there, see:

- https://confident.dbis.rwth-aachen.de/dblpconf/series/webist
- https://confident.dbis.rwth-aachen.de/dblpconf/series/itsc

Having more data as a starting point in OpenResearch seems like a good idea to me.

**Reply by WolfgangFahl** [reply]
I love it ...

**Comment by fschwinger** [comment]
For clarification: During the doctoral seminar at i5, after Wolfgang's talk, I searched for conferences in my area on OpenResearch, where I was planning on submitting papers. Unfortunately, none of the conferences I searched for where listed on there, which is why I think that the project would be more useful, if there is already a conference corpus in the system, which then can of course be extended by the research community.

Furthermore, the search starting point from a paper would also help me on identifying relevant conference events. Often I try to find relevant venues by looking at past papers with a large impact and looking where the authors have published their papers. Therefore, having links from individual papers to a respective conference event and from there to the conference series would be a great addition!

**Reply by WolfgangFahl** [reply]
see #112 for a prototype how to go from a paper to the event.

**Reply by clange** [reply]
I confirm from my own experience that **publications form an important entry point into the domain of events**. In other words, that the "shortest" way for a researcher to add a new event to OpenResearch is indeed likely to start from the researcher's publication list. It took me time to realize this, because I had always believed in _events_ as first-class citizens.
Nevertheless: pretty much every researcher has a well maintained _publication list_ as a part of their academic CV, and there are tools (e.g., Google Scholar), for maintaining it. I am not aware of a widely used tool for maintaining "the list of events I have attended". (Well, OpenResearch could become that.)
